x/oracle/abci.go

```diff
@@ -57,7 +57,14 @@ func CalcPrices(ctx sdk.Context, params types.Params, k keeper.Keeper) error {
 
 	// Iterate through ballots and update exchange rates; drop if not enough votes have been achieved.
 	for _, ballotDenom := range ballotDenomSlice {
+		// Convert ballot power to a percentage to compare with VoteThreshold param
+		if sdk.NewDecWithPrec(ballotDenom.Ballot.Power(), 2).LTE(k.VoteThreshold(ctx)) {
+			ctx.Logger().Info("Ballot voting power is under vote threshold, dropping ballot for", ballotDenom)
```

Comment:
I believe the last two args here are supposed to act like a key value pair
```suggestion
			ctx.Logger().Info("Ballot voting power is under vote threshold, dropping ballot", "denom", ballotDenom)
```